src/components/kytos/misc/ContextPanel.vue

```diff
@@ -15,15 +17,44 @@ import KytosBaseWithIcon from '../base/KytosBaseWithIcon';
 
 /**
  * Represents a context where the developer can add any desired content.
+ *
+ * @example:
+ * <k-context-panel title_color="#554077" subtitle="Some Subtitle" title="Some Title" icon="gear">
+ *     <any-k-component></any-k-component>
+ * </k-context-panel>
+ *
+ * @param {string} title: Title of the k-context-panel
+ * @param {string} title_color: Define custom color of the title
+ *                      ** Hexadecimal for non CSS standard colors, but it is strongly
+ *                         recommended the use of our kytos-colors. **
+ * @param {string} icon: Define custom icon to k-context-panel
+ * @param {string} subtitle: Subtitle of k-context-panel
  */
 export default {
   name: 'k-context-panel',
   mixins: [KytosBaseWithIcon],
   props: {
+    title: {
+      type: String
+    },
+    icon: {
+      type: String
+    },
     subtitle: {
-        type: String
+      type: String
+    },
+    title_color: {
+      type: String
     },
   },
+  computed: {
+    style() {
+      let color
+      (this.title_color !== undefined) ? color = 'color:' + this.title_color : color = 'color: #CCC'
```

Comment:
Maybe more parenthesis would make this a little easier to understand?
```suggestion
      (this.title_color !== undefined) ? (color = 'color:' + this.title_color) : (color = 'color: #CCC')
```

Or just turn that into a regular if/else to make things easier for the reader.